python/ngraph/ops.py

```diff
@@ -951,6 +951,55 @@ def lrn(data,       # type: Node
     return LRN(data, alpha, beta, bias, size)
 
 
+@nameable_op
+def argmax(data,    # type: Node
+           axis,    # type: int
+           etype,   # type: NumericType
+           ):
+    # type: (...) -> Node
+    """return a node which performs ArgMax index reduction operation.
+    :param data: Input data.
+    :param axis: Reduction Axis.
+    :param etype: Index element type.
+    :return: The new node which performs ArgMax
+    """
+    return ArgMax(data, axis, get_element_type(etype))
+
+
+@nameable_op
+def argmin(data,    # type: Node
+           axis,    # type: int
+           etype,   # type: NumericType
+           ):
+    # type: (...) -> Node
+    """return a node which performs ArgMin index reduction operation.
+    :param data: Input data.
+    :param axis: Reduction Axis.
+    :param etype: Index element type.
+    :return: The new node which performs ArgMin
+    """
+    return ArgMin(data, axis, get_element_type(etype))
+
+
+@nameable_op
+def topk(data,    # type: Node
+         kaxis,   # type: int
+         etype,   # type: NumericType
+         k,       # type: int
+         cmax     # type: bool
+         ):
+    # type: (...) -> Node
+    """return a node which performs TopK.
+    :param data: Input data.
+    :param kaxis: TopK Axis.
+    :param etype: Index element type.
+    :param k: K.
+    :param cmax: Compute TopK largest or smallest?
+    :return: The new node which performs ArgMin
```

Comment:
`The new node which performs ArgMin` -> `The new node which performs TopK`
What does TopK produce? Values or indices of values? Please add to docs.